src/dual-alb-fargate-svc.ts

```diff
@@ -64,6 +64,9 @@ export class DualAlbFargateService extends BaseFargateService {
           targetGroup: exttg,
         });
         exttg.addTarget(this.service[index]);
+        if (t.healthCheckConfigure) {
+          exttg.configureHealthCheck({ ...t.healthCheckConfigure });
+        };
```

Comment:
```suggestion
        if (t.healthCheck) {
          exttg.configureHealthCheck({ ...t.healthCheck });
        };
```

Let's just name it `healthCheck`